src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
+              if (!options.useContentExp) {
```

Comment:
`options.useContentExp` is used when the attribute *never changes* because it's actually an expression such as: `tooltipTemplate="'someUrl.html'"`. In such cases, the `tooltipTemplate` attribute never changes (it's always `'someUrl.html'`). So I'm not sure why you would need to run positionTooltip if `options.useContentExp` is true.